packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -103,20 +104,47 @@ module.exports = (
         auth: {
           strategy: 'sessionToken',
         },
+        description: '🔒 sessionToken',
+        notes: [
+          dedent`
+            🔒 HAWK-authenticated with session token
+
+            Returns the 'verified' status for the account's recovery email address.
+
+            Currently, each account is associated with exactly one email address. This address must be verified before the account can be used (specifically, \`POST /certificate/sign\` and \`GET /account/keys\` will return errors until the address is verified). In the future, this may be expanded to include multiple addresses, and/or alternate types of recovery methods (e.g. SMS). A new API will be provided for this extra functionality.
```

Comment:
We just ripped out SMS, so I think we can take that out. I also don't think "Currently, each account is associated with exactly one email address." is necessarily accurate since we have secondary emails, so this entire paragraph might need to be tweaked.

Also, we have "HAWK-authenticated with session token" on a lot of this, but... we deprecated HAWK [almost two years ago](https://github.com/mozilla/fxa/blob/main/docs/adr/0022-deprecate-hawk.md). I'm not really sure what that means for all of these routes, it might mean they're optionally HAWK-authenticated since the GQL server doesn't provide HAWK-auth AFAIK. Might ask @bbangert, and it might be good for him to take a gander at this PR as well before it's merged.

Reply:
Looks like a lot of these descriptions were taken from our existing docs which makes sense. I don't think this needs to be changed before r+, but it can certainly go in a follow up issue on just reviewing these docs and adding to them where we're missing.